swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -0,0 +1,157 @@
+// Ready translated locale messages
+
+const translations = {
+  en: {
+    message: {
+      index: {
+        logIn: "Log In",
+      },
+      error: {
+        frontPage: "To the Front Page",
+        Unauthorized: "401 – Not logged in",
+        Unauthorized_text: "The action you want to take requires logging" +
+                           "in. You can do so with the button below.",
```

Comment:
```suggestion
                           "in. Use the Log In button below.",
```

preferably not to use 2nd person in texts e.g. (You) - it sounds like we are blaming the user for doing something wrong.